test/operator.jl

```diff
@@ -586,6 +590,25 @@ const sub2 = JuMP.repl[:sub2]
                                          x[2] + 3x[3]])
 
         @test vec_eq(@JuMP.Expression(A*x/2), A*x/2)
+        @test vec_eq(X*v,  [4X11; 6X23; 0])
+        @test vec_eq(v'*X, [4X11  0     5X23])
+        @test vec_eq(X*A,  [2X11  X11  0
+                            0     X23  2X23
+                            0     0    0   ])
+        @test vec_eq(A*X,  [2X11  0    X23
+                            X11   0    2X23
+                            0     0    X23])
+        @test vec_eq(A*X', [2X11  0    0
+                            X11   X23  0
+                            0     2X23 0])
+        @test vec_eq(X'*A, [2X11  X11  0
+                            0     0    0
+                            X23   2X23 X23])
+        @test vec_eq(X*A, X*B)
+        @test vec_eq(A*X, B*X)
+        @test vec_eq(A*X', B*X')
+        # TODO this is known to fail currently
+        @test vec_eq(X'*A, X'*B)
```

Comment:
Done.  There is an error in the concatenation section, but I don't think it's new.